helm_deploy/values-dev.yaml

```diff
@@ -1,38 +1,27 @@
-# This is a YAML-formatted file.
-# Declare variables to be passed into your templates.
+---
+# Environment specific values, override helm_deploy/hmpps-manage-users/values.yaml
 
-replicaCount: 2
+generic-service:
+  replicaCount: 2
+  ingress:
+    host: manage-users-dev.hmpps.service.justice.gov.uk
+    modsecurity_audit_enabled: true
 
-image:
-  repository: quay.io/hmpps/hmpps-manage-users
-  tag: latest
-  pullPolicy: IfNotPresent
-  port: 3000
+  env:
+    API_ENDPOINT_URL: https://api-dev.prison.service.justice.gov.uk
+    MANAGE_USERS_API_ENDPOINT_URL: https://hmpps-manage-users-api-dev.hmpps.service.justice.gov.uk
+    OAUTH_ENDPOINT_URL: https://sign-in-dev.hmpps.service.justice.gov.uk/auth
+    DPS_ENDPOINT_URL: https://digital-dev.prison.service.justice.gov.uk
+    HMPPS_COOKIE_NAME: hmpps-session-dev
+    NODE_ENV: production
```

Comment:
some of these look identical across envs, so would be better in `values.yaml` instead